src/annotator/anchoring/pdf.js

```diff
@@ -330,7 +330,8 @@ function findInPages(pageIndexes, quoteSelector, positionHint) {
   const offset = getPageOffset(pageIndex);
 
   const attempt = ([content, offset]) => {
-    const root = { textContent: content };
+    const root = document.createElement('div');
```

Comment:
Is there any meaningful performance impact here?

Reply:
This will be slightly more expensive but the impact should be marginal in the context of everything else going on. I'll do some more testing with heavily annotated pages.